src/main/java/com/flightstats/hub/webhook/TimedWebhookStrategy.java

```diff
@@ -187,8 +187,10 @@ private void doWork() throws InterruptedException {
                     nextTime = lastAdded.getTime();
                 }
                 DateTime stable = TimeUtil.stable().minus(duration);
-                Optional<ChannelConfig> optionalChannelConfig = channelService.getChannelConfig(channel, true);
-                if (optionalChannelConfig.isPresent() && !optionalChannelConfig.get().isLive()) {
+                boolean notLiveChannel = channelService.getChannelConfig(channel, true)
```

Comment:
I dunno if this'd be a worthwhile method to extract over to the channelService here, but we use it in a couple places, so it might be a nice bit of tidying up.

Reply:
hmm, this is also backwards, it should filter OUT the "live" channel 👎 glad that you mentioned something